packages/react/src/useTableUtils/helpers.ts

```diff
@@ -135,6 +135,30 @@ export const getTableRows = (spec: TableSpec, records: GadgetRecord<any>[]) => {
   return records.map((record) => _recordToRow(spec, record));
 };
 
+const getColumnRelatedField = (targetColumn: string | RelatedFieldColumn | CustomCellColumn, field: FieldMetadataFragment) => {
```

Comment:
I'm confused why this ended up in here?

Reply:
Good catch. This was a leftover from getting through a merge conflict